dist/src/main/java/io/camunda/zeebe/broker/BrokerModuleConfiguration.java

```diff
@@ -137,6 +138,15 @@ private void cleanupWorkingDirectory() {
     }
   }
 
+  private void prometheusRegistrySettings(final PrometheusMeterRegistry meterRegistry) {
+    meterRegistry
+        .config()
+        .onMeterRegistrationFailed(
+            (id, error) -> {
+              LOGGER.warn("Failed to register metric {}: {}", id, error);
+            });
+  }
```

Comment:
:wrench: My understanding of Spring is this kind of code should go in a general configuration component. I assume here it's because you didn't want to apply it to the whole stack, right? Because a test fails? So if we fix the issue in tasklist, we can just apply this as a configuration component for everyone?

Reply:
This part has been moved to #24780